src/main/kotlin/uk/gov/dluhc/printapi/database/repository/CertificateRepository.kt

```diff
@@ -17,4 +18,10 @@ interface CertificateRepository : JpaRepository<Certificate, UUID> {
     fun findByStatus(certificateStatus: Status): List<Certificate>
 
     fun findByGssCodeInAndSourceTypeAndSourceReference(gssCodes: List<String>, sourceType: SourceType, sourceReference: String): Certificate?
+
+    fun findByPrintRequestsStatusHistoryEventDateTimeBetweenAndPrintRequestsStatusHistoryStatus(
```

Comment:
Do we need to find the certificates or just get a `count`?  (I think for the rate limit just need a count)

Reply:
Thanks, @Neil-Massey, for the suggestions. 
I definitely agree with you. 
I will make the changes accordingly.